obspy/io/sac/core.py

```diff
@@ -381,7 +381,12 @@ def _internal_read_sac(buf, headonly=False, debug_headers=False, fsize=True,
     :return: A ObsPy Stream object.
```

Comment:
That's very nice of you. I will go ahead and change the rest of the PR and make a push now.